project-generated-code-backend/Backend/Model/Hospital/Building.cs

```diff
@@ -5,9 +5,9 @@ namespace health_clinic_class_diagram.Backend.Model.Hospital
 {
     public class Building : Entity
     {
-        public string Id { get; }
+        public string Id { get; set;  }
         public string Name { get; set; }
-        public List<Floor> Floors { get; set; }
+        public string Floors { get; set; }
```

Comment:
Should this be a list? If it only contains ids, a better name would be FloorIds

Reply:
Noted! This will be changed in backend.